File: labs/solidworks/swex/pmpage/controls/option-box/OptionBox.cs
using CodeStack.SwEx.Common.Attributes;
using CodeStack.SwEx.PMPage.Attributes;

public class OptionBoxDataModel
{
    public enum Options_e
    {
        Option1,
        Option2,
        [Title("Third Option")]
        Option3
    }

    [OptionBox]
    public Options_e Options { get; set; }
}

File: labs/solidworks/swex/pmpage/controls/option-box/OptionBox.vb
Imports CodeStack.SwEx.Common.Attributes
Imports CodeStack.SwEx.PMPage.Attributes

Public Class OptionBoxDataModel

    Public Enum Options_e
        Option1
        Option2
        <Title("Third Option")>
        Option3
    End Enum

    <OptionBox>
    Public Property Options As Options_e

End Class
